### DISCUSSION TITLE
Can I export merged layers?

### DISCUSSION THREAD

**Discussion by maribox** [khalim19/gimp-plugin-export-layers#111] [Q&A]
So I want to export an animation that builds on top of each other as separate images.
For example:
I have 4 Layers:

one dot on the right
one dot in the middle
one dot on the left
1 White Background

Now what I want is an animation that looks like an empty sheet, then one dot, then two, then three. So my export should be:
{Layer 1}, {Layer 1 + Layer 2}, {Layer 1 + Layer 2 + Layer 3}, {Layer 1 + Layer 2 + Layer 3 + Layer 4}, kind of as if I activated layers one after the other. 
Does that exist and if no might that not be a useful feature for a lot of animations that build on top of the previous layers?

**Comment by kamilburda** [comment]
Currently, the only way you could make this work is you export a single image in a format supporting animations, e.g. animated GIF. You could do this as follows:
1. Mark the white background layer with the `Background` tag (right-click in the preview on the layer name)
2. Add the `Insert background layers` procedure
3. Add the `Export` procedure, adjust the file extension and set `Perform export:` to `For the entire image at once`

If you intend to export the layers individually, then this is currently not possible. I'll see what I can do, though, I can see this being implemented easily (by adding an option to not remove layers at the end of processing).
